### ISSUE TITLE
Missing support for .gitattributes files

### ISSUE THREAD

**Issue by djanosik** [madskristensen/TextmateBundleInstaller#8]
VS Code supports syntax highlighting for .gitattributes. It's missing in VS 2015.

**Comment by madskristensen** [comment]
I can't find a Textmate bundle for .gitattributes. Do you know of one?

**Comment by djanosik** [comment]
I see that VS Code uses highlighter for .ini files. Can you try?

**Comment by madskristensen** [comment]
Great, I added .gitattributes to the existing .ini textmate bundle. I'll release an update asap

**Comment by madskristensen** [comment]
Grab version 1.3.30 from here https://visualstudiogallery.msdn.microsoft.com/d92fd742-bab3-4314-b866-50b871d679ee to test it out

**Comment by djanosik** [comment]
It feels broken :/
```
20.09.2016 19:38:35 - Install Error : System.ArgumentException: Illegal characters in path.
   at System.IO.Path.CheckInvalidPathChars(String path, Boolean checkAdditional)
   at System.IO.Path.Combine(String path1, String path2)
   at Microsoft.VisualStudio.ExtensionManager.ExtensionManagerService.WriteFilesToInstallDirectory(InstallableExtensionImpl extension, String installPath, ZipPackage vsixPackage, IDictionary`2 extensionsInstalledSoFar, AsyncOperation asyncOp, UInt64 totalBytesToWrite, UInt64& totalBytesWritten)
   at Microsoft.VisualStudio.ExtensionManager.ExtensionManagerService.InstallInternal(InstallableExtensionImpl extension, Boolean perMachine, Boolean isNestedExtension, IDictionary`2 extensionsInstalledSoFar, List`1 extensionsUninstalledSoFar, IInstalledExtensionList modifiedInstalledExtensionsList, AsyncOperation asyncOp, UInt64 totalBytesToWrite, UInt64& totalBytesWritten, Boolean enforceCertificateCheckForUpgrade)
   at Microsoft.VisualStudio.ExtensionManager.ExtensionManagerService.BeginInstall(IInstallableExtension installableExtension, Boolean perMachine, AsyncOperation asyncOp, Boolean enforceCertificateCheckForUpgrade)
   at Microsoft.VisualStudio.ExtensionManager.ExtensionManagerService.InstallWorker(IInstallableExtension extension, Boolean perMachine, AsyncOperation asyncOp, Boolean enforceCertificateCheckForUpgrade)
```

**Comment by madskristensen** [comment]
Yeah, just saw that. Fix on its way...

**Comment by madskristensen** [comment]
Fixed in version 1.3.32

**Comment by djanosik** [comment]
Thank you! It works.
